src/main/java/UI/ListingListPage.java

```diff
@@ -1,36 +1,34 @@
 package UI;
 import entities.Listing;
+import database.DatabaseController;
 
+import java.awt.*;
 import java.util.List;
+import javax.swing.JFrame;
+import javax.swing.*;
 
 public class ListingListPage extends Page {
-    private List<Listing> listings;
 
-    // super call from page, assume every page title will be listings?
-    public ListingListPage(List<Listing> listings) {
-        super("Listings");
-        this.listings = listings;
+
+    public ListingListPage(String title) {
+        super(title);
+        JFrame container = new JFrame();
```

Comment:
Should this be a JPanel?

Reply:
disregard this, it wasn't supposed to be on this branch